site/content/docs/5.0/components/modal.md

```diff
@@ -509,6 +509,37 @@ exampleModal.addEventListener('show.bs.modal', function (event) {
 })
 ```
 
+### Toggle dialogs
+
+```html
```

Comment:
You need to check the examples above and make this a proper example. Right now, only the snippet shows.

Reply:
@XhmikosR you mean add description/hint to snippet? could you explain you answer?

Reply:
@XhmikosR if problem in description, then I need help - as you can see, my English is not so good.

Reply:
It's not the description, it's the whole example. Please check the preview and see how this looks compared to the existent examples.

Reply:
@XhmikosR what's difference between my example and that? Can you post standard/pattern of  "proper example"?
~~~html
<!-- Scrollable modal -->
<div class="modal-dialog modal-dialog-scrollable">
  ...
</div>
~~~

Reply:
@WinterSilence He means that your additions here only will show the snippet of code, not a rendered interactive example in our docs. He's asking you to review the other examples in this file because they'll show you the syntax we use to do that.

Reply:
@mdo update, you told about it?